python/hail/utils/java.py

```diff
@@ -127,11 +127,19 @@ def numpy_from_breeze(bdm):
     if bdm.majorStride() != expected_stride:
         raise ValueError("Expected major stride of Breeze matrix to be {}, found {}"
                          .format(expected_stride, bdm.majorStride()))
-    if cols * rows > 0x7ffff:
+    if cols * rows > 0x7ffffff:
         raise ValueError("rows * cols must be smaller than {}, found {} by {} matrix"
-                         .format(0x7ffff, rows, cols))
+                         .format(0x7ffffff, rows, cols))
 
-    b = Env.jutils().bdmToBytes(bdm)
+    if cols * rows <= 0x7ffff:
```

Comment:
why did you choose 0x7ffff?

Reply:
Maxint/8

Reply:
that has floor 0xFFFFFFF

Reply:
oops, you're right. I shifted by 3 x 4 bits instead of by 3 bits.